python/starring_them_all.py

```diff
@@ -0,0 +1,21 @@
+import requests
+
+# GitHub organization name
+org_names = ['brainglobe', 'neuroinformatics-unit']
+
+import subprocess
```

Comment:
```suggestion
"""A rough script to star all our repos, because they are awesome (and there are many of them!)
"""
import requests
import subprocess

org_names = ['brainglobe', 'neuroinformatics-unit'] # add/remove other orgs you'd like to star here
```
Tiny tidy-up/documenting